### PR TITLE
fix: Prevent pgm_conf not found prompt

### PR DESCRIPTION
**Pull request by jirimosinger**

### CODE REVIEW

File: clients/vscode-hlasmplugin/src/configurationsHandler.ts [hunk 1]
```diff
@@ -68,7 +63,7 @@ export class ConfigurationsHandler {
                         ConfigurationsHandler.createProcTemplate(workspace).then(uri => vscode.commands.executeCommand("vscode.open", uri));
                     }
                 });
-        if (!p.exists && !e.exists)
+        if (!p.exists && !b.exists && !e.exists)
```
Comment (slavek-kucera) [inline clients/vscode-hlasmplugin/src/configurationsHandler.ts:66]: Maybe a topic for another PR - convert this to the "smarter" variant that is used in the code actions?

File: clients/vscode-hlasmplugin/src/test/mocks.ts [hunk 2]
```diff
@@ -248,3 +248,50 @@ export class TextDocumentMock implements vscode.TextDocument {
             return new vscode.Position(0, this.text.length);
     }
 }
+
+export class FileSystemMock implements vscode.FileSystem {
+    resourceUris = new Set<string>();
+    dummyStat: vscode.FileStat;
+
+    public addUrifsPair(uri: vscode.Uri) {
```
Comment (slavek-kucera) [inline clients/vscode-hlasmplugin/src/test/mocks.ts:256]: what `Pair`?

**Review by slavek-kucera — APPROVED** [review]

File: clients/vscode-hlasmplugin/src/test/mocks.ts (hunk 2 above)
Comment (jirimosinger) [reply clients/vscode-hlasmplugin/src/test/mocks.ts]: Good catch!

File: clients/vscode-hlasmplugin/src/configurationsHandler.ts (hunk 1 above)
Comment (jirimosinger) [reply clients/vscode-hlasmplugin/src/configurationsHandler.ts:66]: Sure, I can look into that afterwards

**Review by slavek-kucera — APPROVED** [review]

**Pull request merged**